indico_zodbimport/modules/event_layout.py

```diff
@@ -83,12 +86,23 @@ def migrate_layout_settings(self):
                                      event_id=event.id)
                     continue
                 with open(path, 'rb') as f:
-                    logo_content = f.read()
-                logo_content_type = mimetypes.guess_type(logo.fileName)[0]
+                    try:
+                        logo_image = Image.open(f)
```

Comment:
only this line should be in the `try` block (or does it ever fail somewhere else with an IOError?)